rust/optimized/main.rs

```diff
@@ -6,6 +6,7 @@
 // There's nothing particularly interesting here other than swapping out std's
 // default hashing algorithm for one that isn't cryptographically secure.
 
+use bumpalo::Bump;
```

Comment:
Third party imports should be grouped together. I'd put this right above the `fxhash` comment below.

Reply:
Done